chain/client/src/tests/catching_up.rs

```diff
@@ -694,7 +694,6 @@ enum ChunkGrievingPhases {
 
 // TODO(#3180): seals are disabled in single shard setting
 #[test]
```

Comment:
is it related to this PR?

Reply:
Yes: new version of the compiler contains a bug fix which makes it correctly emit a warning about duplicate `ignore`